src/Requests/Transfers24.php

```diff
@@ -623,6 +635,7 @@ public function setFields()
         $this->setField('p24_price_1', $this->article_price);
         $this->setField('p24_number_1', $this->article_number);
         $this->setField('p24_shipping', $this->shipping_cost);
+        $this->setField('p24_encoding', $this->encoding);
```

Comment:
It should be set only when encoding was set, otherwise this field shouldn't be sent